### DISCUSSION TITLE
name 'false' is not defined?

### DISCUSSION THREAD

**Discussion by Ulash4222** [mrdbourke/pytorch-deep-learning#671] [Q&A]
Hi,
In Chapter 4 Video 80, i encountered with this error. I looked it up however i could not see any discussions about it. Or maybe i just did something wrong. I add the error below. Thank you for answers. 

---
helper_functions.py already exists, skipping download
---------------------------------------------------------------------------
NameError                                 Traceback (most recent call last)
[<ipython-input-99-cc814271754b>](https://localhost:8080/#) in <cell line: 13>()
     11     f.write(request.content)
     12 
---> 13 from helper_functions import plot_predictions, plot_decision_boundary

[/content/helper_functions.py](https://localhost:8080/#) in <module>
----> 1 {"payload":{"allShortcutsEnabled":false,"fileTree":{"":{"items":[{"name":".github","path":".github","contentType":"directory"},{"name":"data","path":"data","contentType":"directory"},{"name":"demos","path":"demos","contentType":"directory"},{"name":"docs","path":"docs","contentType":"directory"},{"name":"extras","path":"extras","contentType":"directory"},{"name":"going_modular","path":"going_modular","contentTyp...

NameError: name 'false' is not defined

**Comment by shisirkha** [comment]
It would be False not false because Python `True` and `False` are not same as true, false

the difference between these 2 examples is that one uses first capital that is valid in python and another example uses all lower case which is not defined so the python thinks false as a variable. (that is not defined)

so, the actual way of writing true and false is `True,False`.

also, don't feel guilty when seeing that this was a small mistake, and you can't fix it. 
everyone makes mistakes.

I also make stupid mistakes much.
